### PR TITLE
Migrate Blackbaud employee plugin during eject

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

File: lib/eject.js [hunk 1]
```diff
@@ -350,7 +350,7 @@ async function eject() {
 
     // Setup routes.
     const routes = routesGenerator.getRoutesData(skyuxConfig);
-    createRouteComponentFiles(ejectedProjectPath, routes)
```
Comment (Blackbaud-PaulCrowder) [inline lib/eject.js:353]: This is an unrelated change; I just noticed a semicolon was missing.

**Review by Blackbaud-SteveBrush — APPROVED** [review]
